src/Iterators/FilteredRecursiveDirectoryIterator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace PHP_Parallel_Lint\PhpParallelLint\Iterators;
+
+use ReturnTypeWillChange;
+
+class FilteredRecursiveDirectoryIterator extends \RecursiveDirectoryIterator
+{
+    private $excluded = array();
+
+    public function __construct($path, $flags = null, array $excluded = array())
+    {
+        $this->excluded = $excluded;
+
+        if ($flags === null) {
+            $flags = \FilesystemIterator::KEY_AS_PATHNAME | \FilesystemIterator::CURRENT_AS_FILEINFO;
+        }
+
+        parent::__construct($path, $flags);
+    }
+
+    #[ReturnTypeWillChange]
+    public function hasChildren($allowLinks = false)
+    {
+        if (in_array($this->getBasename(), $this->excluded)) {h
```

Comment:
```suggestion
        if (in_array($this->getBasename(), $this->excluded)) {
```

Looks like a stray parse error made it into the patch ?